[2024-03-12T21:22:11.824-0400] {processor.py:161} INFO - Started process (PID=407069) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:22:11.827-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:22:11.828-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:22:11.828-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:22:11.835-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:22:11.834-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:22:11.836-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:22:11.835-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:22:11.836-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:22:11.858-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-03-12T21:24:03.782-0400] {processor.py:161} INFO - Started process (PID=407601) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:24:03.835-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:24:03.838-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:24:03.837-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:24:03.844-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:24:03.843-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:24:03.844-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:24:03.844-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:24:03.845-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:24:03.885-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.107 seconds
[2024-03-12T21:26:58.157-0400] {processor.py:161} INFO - Started process (PID=408350) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:26:58.208-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:26:58.210-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:26:58.209-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:26:58.218-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:26:58.216-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:26:58.218-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:26:58.218-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:26:58.219-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:26:58.237-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2024-03-12T21:28:37.738-0400] {processor.py:161} INFO - Started process (PID=408837) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:28:37.743-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:28:37.750-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:28:37.749-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:28:37.760-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:28:37.759-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:28:37.761-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:28:37.760-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:28:37.761-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:28:37.787-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-03-12T21:30:04.771-0400] {processor.py:161} INFO - Started process (PID=409256) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:30:04.822-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:30:04.825-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:30:04.824-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:30:04.834-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:30:04.832-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:30:04.835-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:30:04.834-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:30:04.836-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:30:04.860-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2024-03-12T21:31:44.761-0400] {processor.py:161} INFO - Started process (PID=409751) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:31:44.765-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:31:44.767-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:31:44.766-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:31:44.775-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:31:44.774-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:31:44.776-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:31:44.776-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:31:44.777-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:31:44.800-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-03-12T21:33:25.777-0400] {processor.py:161} INFO - Started process (PID=410310) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:33:25.781-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:33:25.784-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:33:25.783-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:33:25.794-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:33:25.793-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:33:25.795-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:33:25.795-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:33:25.796-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:33:25.822-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-03-12T21:34:55.148-0400] {processor.py:161} INFO - Started process (PID=410721) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:34:55.199-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T21:34:55.202-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:34:55.200-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:34:55.215-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:34:55.213-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T21:34:55.217-0400] {logging_mixin.py:188} INFO - [2024-03-12T21:34:55.216-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T21:34:55.217-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T21:34:55.239-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2024-03-12T23:26:32.292-0400] {processor.py:161} INFO - Started process (PID=417770) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:26:32.344-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:26:32.347-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:26:32.345-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:26:32.358-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:26:32.356-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:26:32.361-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:26:32.361-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:26:32.362-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:26:32.391-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.108 seconds
[2024-03-12T23:29:45.626-0400] {processor.py:161} INFO - Started process (PID=418546) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:29:45.644-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:29:45.647-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:29:45.646-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:29:45.656-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:29:45.654-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:29:45.659-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:29:45.656-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:29:45.660-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:29:45.682-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.061 seconds
[2024-03-12T23:31:34.165-0400] {processor.py:161} INFO - Started process (PID=419011) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:31:34.220-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:31:34.223-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:31:34.222-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:31:34.241-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:31:34.239-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:31:34.241-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:31:34.241-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:31:34.242-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:31:34.282-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.123 seconds
[2024-03-12T23:42:29.692-0400] {processor.py:161} INFO - Started process (PID=421626) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:42:29.742-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:42:29.744-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:42:29.744-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:42:29.750-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:42:29.749-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:42:29.750-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:42:29.750-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:42:29.751-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:42:29.769-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2024-03-12T23:44:29.822-0400] {processor.py:161} INFO - Started process (PID=422129) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:44:29.872-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:44:29.874-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:44:29.873-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:44:29.881-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:44:29.880-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:44:29.882-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:44:29.882-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:44:29.883-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:44:29.916-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.099 seconds
[2024-03-12T23:46:09.759-0400] {processor.py:161} INFO - Started process (PID=422709) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:46:09.762-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:46:09.774-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:46:09.763-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:46:09.783-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:46:09.782-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:46:09.784-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:46:09.784-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:46:09.795-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:46:09.840-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.096 seconds
[2024-03-12T23:48:19.153-0400] {processor.py:161} INFO - Started process (PID=423323) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:48:19.205-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:48:19.209-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:48:19.207-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:48:19.217-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:48:19.216-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:48:19.218-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:48:19.218-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:48:19.218-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:48:19.248-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.104 seconds
[2024-03-12T23:49:48.196-0400] {processor.py:161} INFO - Started process (PID=423725) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:49:48.201-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:49:48.205-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:49:48.204-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:49:48.213-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:49:48.212-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:49:48.214-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:49:48.213-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:49:48.215-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:49:48.245-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2024-03-12T23:51:13.846-0400] {processor.py:161} INFO - Started process (PID=424177) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:51:13.898-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:51:13.901-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:51:13.900-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:51:13.908-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:51:13.907-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:51:13.908-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:51:13.908-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:51:13.909-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:51:13.928-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2024-03-12T23:52:41.278-0400] {processor.py:161} INFO - Started process (PID=424648) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:52:41.330-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:52:41.333-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:52:41.332-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:52:41.340-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:52:41.339-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:52:41.340-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:52:41.340-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:52:41.342-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:52:41.367-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.095 seconds
[2024-03-12T23:54:15.806-0400] {processor.py:161} INFO - Started process (PID=425148) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:54:15.862-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:54:15.865-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:54:15.864-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:54:15.873-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:54:15.872-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:54:15.877-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:54:15.875-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:54:15.877-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:54:15.905-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.105 seconds
[2024-03-12T23:55:54.031-0400] {processor.py:161} INFO - Started process (PID=425574) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:55:54.034-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-12T23:55:54.038-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:55:54.035-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:55:54.046-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:55:54.046-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-12T23:55:54.047-0400] {logging_mixin.py:188} INFO - [2024-03-12T23:55:54.047-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-12T23:55:54.047-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-12T23:55:54.067-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.048 seconds
[2024-03-13T00:30:19.253-0400] {processor.py:161} INFO - Started process (PID=432883) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:30:19.306-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:30:19.309-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:30:19.308-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:30:19.318-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:30:19.316-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:30:19.318-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:30:19.318-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:30:19.320-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:30:19.346-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.100 seconds
[2024-03-13T00:32:08.945-0400] {processor.py:161} INFO - Started process (PID=433393) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:32:08.995-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:32:08.998-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:32:08.997-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:32:09.004-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:32:09.003-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:32:09.004-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:32:09.004-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:32:09.005-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:32:09.025-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2024-03-13T00:36:03.005-0400] {processor.py:161} INFO - Started process (PID=434357) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:36:03.009-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:36:03.011-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:36:03.010-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:36:03.017-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:36:03.016-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:36:03.018-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:36:03.018-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:36:03.019-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:36:03.046-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2024-03-13T00:38:14.594-0400] {processor.py:161} INFO - Started process (PID=435006) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:38:14.598-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:38:14.600-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:38:14.599-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:38:14.605-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:38:14.605-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:38:14.606-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:38:14.606-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:38:14.606-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:38:14.624-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-03-13T00:39:51.534-0400] {processor.py:161} INFO - Started process (PID=435433) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:39:51.581-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:39:51.584-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:39:51.583-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:39:51.593-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:39:51.590-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:39:51.594-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:39:51.593-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:39:51.595-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:39:51.617-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.092 seconds
[2024-03-13T00:41:15.888-0400] {processor.py:161} INFO - Started process (PID=435817) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:41:15.892-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:41:15.893-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:41:15.893-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:41:15.899-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:41:15.898-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:41:15.899-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:41:15.899-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:41:15.900-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:41:15.915-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-03-13T00:44:20.051-0400] {processor.py:161} INFO - Started process (PID=436659) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:44:20.101-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:44:20.103-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:44:20.103-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:44:20.113-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:44:20.111-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:44:20.114-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:44:20.113-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:44:20.114-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:44:20.131-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2024-03-13T00:45:43.061-0400] {processor.py:161} INFO - Started process (PID=437010) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:45:43.111-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T00:45:43.113-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:45:43.112-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:45:43.119-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:45:43.118-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T00:45:43.119-0400] {logging_mixin.py:188} INFO - [2024-03-13T00:45:43.119-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T00:45:43.119-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T00:45:43.137-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2024-03-13T15:49:26.529-0400] {processor.py:161} INFO - Started process (PID=453793) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:49:26.530-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:49:26.531-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:49:26.531-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:49:26.533-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:49:26.533-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:49:26.533-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:49:26.533-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:49:26.533-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:49:26.539-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.012 seconds
[2024-03-13T15:51:35.364-0400] {processor.py:161} INFO - Started process (PID=454519) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:51:35.368-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:51:35.369-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:51:35.368-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:51:35.371-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:51:35.371-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:51:35.371-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:51:35.371-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:51:35.371-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:51:35.379-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-03-13T15:52:30.679-0400] {processor.py:161} INFO - Started process (PID=454810) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:52:30.681-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:52:30.682-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:52:30.682-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:52:30.684-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:52:30.683-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:52:30.684-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:52:30.684-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:52:30.684-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:52:30.691-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-03-13T15:53:24.223-0400] {processor.py:161} INFO - Started process (PID=455099) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:53:24.226-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:53:24.227-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:53:24.227-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:53:24.229-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:53:24.229-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:53:24.229-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:53:24.229-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:53:24.229-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:53:24.236-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-13T15:54:39.004-0400] {processor.py:161} INFO - Started process (PID=455487) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:54:39.008-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:54:39.009-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:54:39.008-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:54:39.011-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:54:39.011-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:54:39.011-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:54:39.011-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:54:39.012-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:54:39.019-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-03-13T15:55:31.873-0400] {processor.py:161} INFO - Started process (PID=455743) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:55:31.875-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:55:31.875-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:55:31.875-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:55:31.878-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:55:31.877-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:55:31.878-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:55:31.878-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:55:31.878-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:55:31.885-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-03-13T15:56:22.345-0400] {processor.py:161} INFO - Started process (PID=456012) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:56:22.347-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:56:22.347-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:56:22.347-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:56:22.349-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:56:22.349-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:56:22.350-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:56:22.350-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:56:22.350-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:56:22.356-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-03-13T15:57:13.941-0400] {processor.py:161} INFO - Started process (PID=456276) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:57:13.944-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:57:13.945-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:57:13.945-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:57:13.947-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:57:13.947-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:57:13.947-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:57:13.947-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:57:13.947-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:57:13.954-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-13T15:58:06.505-0400] {processor.py:161} INFO - Started process (PID=456578) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:58:06.507-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:58:06.508-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:58:06.507-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:58:06.510-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:58:06.509-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:58:06.510-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:58:06.510-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:58:06.510-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:58:06.517-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-13T15:59:22.390-0400] {processor.py:161} INFO - Started process (PID=456931) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:59:22.395-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T15:59:22.397-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:59:22.396-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:59:22.401-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:59:22.401-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T15:59:22.401-0400] {logging_mixin.py:188} INFO - [2024-03-13T15:59:22.401-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T15:59:22.401-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T15:59:22.408-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-03-13T16:00:13.998-0400] {processor.py:161} INFO - Started process (PID=457187) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:00:14.001-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:00:14.002-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:00:14.002-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:00:14.004-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:00:14.004-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:00:14.004-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:00:14.004-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:00:14.004-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:00:14.011-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-13T16:01:29.332-0400] {processor.py:161} INFO - Started process (PID=457594) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:01:29.333-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:01:29.334-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:01:29.334-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:01:29.337-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:01:29.336-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:01:29.337-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:01:29.337-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:01:29.337-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:01:29.344-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-13T16:02:20.427-0400] {processor.py:161} INFO - Started process (PID=457873) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:02:20.429-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:02:20.429-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:02:20.429-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:02:20.432-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:02:20.432-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:02:20.432-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:02:20.432-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:02:20.432-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:02:20.439-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-03-13T16:03:34.473-0400] {processor.py:161} INFO - Started process (PID=458237) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:03:34.474-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:03:34.476-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:03:34.475-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:03:34.478-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:03:34.478-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:03:34.479-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:03:34.478-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:03:34.479-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:03:34.487-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.017 seconds
[2024-03-13T16:04:42.855-0400] {processor.py:161} INFO - Started process (PID=458656) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:04:42.857-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:04:42.857-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:04:42.857-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:04:42.860-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:04:42.859-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:04:42.860-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:04:42.860-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:04:42.860-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:04:42.866-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.013 seconds
[2024-03-13T16:05:53.580-0400] {processor.py:161} INFO - Started process (PID=459022) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:05:53.582-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:05:53.583-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:05:53.583-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:05:53.585-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:05:53.585-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:05:53.586-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:05:53.585-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:05:53.586-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:05:53.593-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-13T16:06:49.328-0400] {processor.py:161} INFO - Started process (PID=459299) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:06:49.332-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:06:49.334-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:06:49.333-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:06:49.341-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:06:49.340-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:06:49.341-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:06:49.341-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:06:49.342-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:06:49.352-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-03-13T16:08:07.137-0400] {processor.py:161} INFO - Started process (PID=459676) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:08:07.140-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:08:07.141-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:08:07.140-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:08:07.143-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:08:07.142-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:08:07.143-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:08:07.143-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:08:07.143-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:08:07.149-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2024-03-13T16:08:57.783-0400] {processor.py:161} INFO - Started process (PID=459933) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:08:57.786-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:08:57.787-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:08:57.787-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:08:57.789-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:08:57.789-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:08:57.789-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:08:57.789-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:08:57.789-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:08:57.796-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
[2024-03-13T16:10:11.850-0400] {processor.py:161} INFO - Started process (PID=460300) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:10:11.854-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:10:11.854-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:10:11.854-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:10:11.856-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:10:11.856-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:10:11.856-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:10:11.856-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:10:11.857-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:10:11.864-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-13T16:11:02.213-0400] {processor.py:161} INFO - Started process (PID=460545) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:11:02.217-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:11:02.218-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:11:02.217-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:11:02.220-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:11:02.220-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:11:02.220-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:11:02.220-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:11:02.220-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:11:02.227-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.015 seconds
[2024-03-13T16:12:16.824-0400] {processor.py:161} INFO - Started process (PID=460913) to work on /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:12:16.828-0400] {processor.py:830} INFO - Processing file /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-03-13T16:12:16.829-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:12:16.828-0400] {dagbag.py:540} INFO - Filling up the DagBag from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:12:16.831-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:12:16.831-0400] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-03-13T16:12:16.831-0400] {logging_mixin.py:188} INFO - [2024-03-13T16:12:16.831-0400] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-03-13T16:12:16.831-0400] {processor.py:842} WARNING - No viable dags retrieved from /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-03-13T16:12:16.838-0400] {processor.py:183} INFO - Processing /home/ironbong_jr/mlops/project/project/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.016 seconds
